README.md

```diff
@@ -17,3 +17,7 @@ That text is a [flipped form](https://flippedform.com/) in everyday English.  If
 - [ ] Compile a registry of projects using the terms.
 
 - [ ] Submit the text to [SPDX](https://spdx.org) for standardization as a license exception.
+
+## Permission
+
+Each contributor licenses you to do everything with the Software Credit Requirement that would otherwise infringe that contributor’s copyright in it.
```

Comment:
```suggestion
Each contributor licenses you to do everything with the legal text that would otherwise infringe that contributor’s copyright in it.

If you make changes to the legal text, you must remove all mention of "Credit Requirement" and the credits.txt project, as well.

***As far as the law allows, the legal text comes as is, without any warranty at all, and no contributor will be liable to anyone for any damages related to the text or its use, for any kind of legal claim.***
```

Reply:
> and the credits.txt project, as well.

Why? It's just an example. For our purposes there's no affiliation here.

Reply:
I'd be fine requiring just a change of name/title. If we change the title, we can always change the license for the license.

Reply:
What license is the license for the license under? :smile: